packages/tools/src/utilities/getModalityUnit.ts

```diff
@@ -1,8 +1,6 @@
 function getModalityUnit(modality: string, isPreScaled: boolean): string {
   if (modality === 'CT') {
     return 'HU';
-  } else if (modality === 'PT' && isPreScaled === true) {
```

Comment:
Why we are removing this unit here?

Reply:
Can we get the metadata and check if suvbw is there in scalingModule, and return 'SUV' to be able to show SUV like before?

Reply:
Good idea. Adapted.